CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-## 8.4.1
-
+## 8.5.0
+* **o-app-layout-sidenav**: implementing 3rd level navigation  ([4daab8d](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/4daab8d)) ([6cbbaa9](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/6cbbaa9)) ([2fc889e](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/2fc889e)) Closes [#709](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/709)
```

Comment:
@patricia-martinez-imatia I think that there might be a breaking change declaration advertising the users that internal HTML structure has changed (switched from _button_ to _a_ and removed _h3_). Maybe some customized css of third applications do not work anymore.